### DISCUSSION TITLE
issue with game instances

### DISCUSSION THREAD

**Discussion by Leon-011** [KSP-CKAN/CKAN#3694] [Q&A] [answered]
i have moved my KSP into a new SSD and now i cant get CKAN to recognize the new file path to KSP......

**Comment by HebaruSan** [comment, marked as answer]
Here's how you add a new game instance:

https://github.com/KSP-CKAN/CKAN/wiki/User-guide#adding-game-folders

**Comment by Veldime13** [comment]
This might be my ignorance with Linux but I'm trying to get the instances to work. So far I have been unable to do so. 
I open the new instance file window but .steam is hidden so I have to grab the link from the Dolphin file explorer.
Pasting the link in the file window for ckan, it gives me an error which I think means it doesn't like the "/" vs. the other slash. I corrected this but the ksp_x64.exe still isn't showing up and the instance isn't found.

Before I post in the bug section, I thought I'd ask here just in case I'm missing something super obvious.
